src/code/object_table.c

```diff
@@ -10,9 +10,20 @@ s16 gPlayerFormObjectIds[PLAYER_FORM_MAX] = {
 
 ObjectId gObjectTableSize = OBJECT_ID_MAX;
 
+// Segment declarations (also used in the table below)
+#define DEFINE_OBJECT(name, _enumValue) DECLARE_ROM_SEGMENT(name)
+#define DEFINE_OBJECT_UNSET(_enumValue)
+#define DEFINE_OBJECT_SIZE_ZERO(_name, _enumValue)
```

Comment:
I think OoT uses `DEFINE_OBJECT_NULL` for these types of entries. Thoughts?

Reply:
Hmm, there rom macro is actually named `EMPTY`, perhaps we should do `DEFINE_OBJECT_EMPTY`?